EC-CUBE/ec-cube
# 【購入画面】支払い方法の手数料が反映されない

## 概要(Overview・Refs Issue)
     【ショップマスター】「商品ごとの送料設定」が機能していない

## 方針(Policy)

## 実装に関する補足(Appendix)

## テスト（Test)
- phpunit
- chrome/firefox

Case 1:
Setup:
- Paymnents Fee:
    + 銀行振込: ￥400
    + 現金書留: ￥50 

Steps:
1. Login
2. Add product to cart
3. Checkout
4. Choose 銀行振込 payment 
    Expected: Payment fee at summary = ￥400
5. Choose 現金書留 payment
    Expected: Payment fee at summary = ￥50
